crates/uv/src/commands/pip/operations.rs

```diff
@@ -97,8 +98,9 @@ pub(crate) async fn resolve<InstalledPackages: InstalledPackagesProvider>(
     options: Options,
     printer: Printer,
     preview: PreviewMode,
+    quiet: bool,
```

Comment:
Could we just pass `Printer::Quiet`?

Reply:
We still want to output the build/download progress to stderr, we just don't want to print the final "Resolved in 20ms" in case we end up needing to resolve again because of warnings.